recipes/grafana-client/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set name = "grafana-client" %}
+{% set version = "3.5.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/grafana-client-{{ version }}.tar.gz
+  sha256: 7ee2f865dbfc89d3eb8bc9df41417ef7c1c905f9d92336bce0af27fb37607bc6
+
+build:
+  skip: true  # [py2k]
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python
+    - setuptools-scm
+    - setuptools-scm <8
+    - pip
+  run:
+    - python
+    - requests >=2.23.0,<3
+    - dataclasses  # [py<=36]
```

```suggestion
    - dataclasses
```
Ref: https://conda-forge.org/docs/maintainer/knowledge_base.html#empty-python-packages